doc/tutorial/conv_arithmetic.txt

```diff
@@ -0,0 +1,597 @@
+
+    \begin{pmatrix}
+    w_{0,0} & 0       & 0       & 0       \\
+    w_{0,1} & w_{0,0} & 0       & 0       \\
+    w_{0,2} & w_{0,1} & 0       & 0       \\
+    0       & w_{0,2} & 0       & 0       \\
+    w_{1,0} & 0       & w_{0,0} & 0       \\
+    w_{1,1} & w_{1,0} & w_{0,1} & w_{0,0} \\
+    w_{1,2} & w_{1,1} & w_{0,2} & w_{0,1} \\
+    0       & w_{1,2} & 0       & w_{0,2} \\
+    w_{2,0} & 0       & w_{1,0} & 0       \\
+    w_{2,1} & w_{2,0} & w_{1,1} & w_{1,0} \\
+    w_{2,2} & w_{2,1} & w_{1,2} & w_{1,1} \\
+    0       & w_{2,2} & 0       & w_{1,2} \\
+    0       & 0       & w_{2,0} & 0       \\
+    0       & 0       & w_{2,1} & w_{2,0} \\
+    0       & 0       & w_{2,2} & w_{2,1} \\
+    0       & 0       & 0       & w_{2,2} \\
+    \end{pmatrix}^T
+
+where :math:`w_{i,j}` is the element of the kernel located at row :math:`i` and
+column :math:`j`. (*Note: the matrix has been transposed for formatting
+purposes.*) This linear operation takes a 16-dimensional vector as input and
+produces a 4-dimensional vector as output.
+
+What would happen if we were to transpose this matrix and feed the corresponding
+linear operation a 4-dimensional vector as input? We would get a 16-dimensional
+vector as output. This is what we call a **transposed convolution**.
+
+Transposed convolutions share the same connectivity pattern as their converse
+convolution, which makes them suitable for decoders in a convolutional
+autoencoder, for instance.
+
+Simplest case, transposed
+=========================
+
+If we visualize the effect of applying the transposed matrix above on a
+:math:`4 \times 4` input, this is what we get:
+
+.. figure:: conv_arithmetic_figures/no_padding_no_strides_transposed.gif
+
+Note that although equivalent to the transposed matrix, this visualization adds
+a lot of zero multiplications in the form of zero padding. This is done for
+illustration purposes, but it is inefficient, and software implementations will
+normally not perform the useless zero multiplications.
+
+One way to understand the logic behind zero padding is to consider what would
+be necessary to produce a transposed convolution with the same connectivity
+pattern as its converse convolution.
+
+In particular, the corner pixels of the input only contribute to the corner
+pixels of the output: the top left input pixel is only connected to the top left
+output pixel, the top right pixel is only connected to the top right output
+pixel, and so on. This is achieved by zero padding the input such that the
+kernel only touches a corner pixel when it is in the associated corner position.
+
+You can also verify that the connectivity pattern is respected everywhere else
+in the input. For instance, when the kernel has been translated by one row and
+one column, it covers the whole input. This means that in the converse
+convolution, the input pixel located at the first row and first column
+contributes to all output pixels.
+
+In general, a convolution described by
```

Comment:
"In general" might be misleading since it's not a rule for the most general case, but rather for the most simple one. I suggest to change it into "In the most simple scenario, i.e. in the case of a convolution described by [...] we can define the transpose as a convolution with these parameters:"